install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -104,6 +104,15 @@ k8sClientRateLimit:
     # @default -- 200
     burst:
 
+# -- Configure exponential backoff for client-go in Cilium agent.
+k8sClientExponentialBackoff:
```

Comment:
Much the same as `k8sClientRateLimit`, should we add a section for the operator here as well? Or is the single operator client too small to matter?

Reply:
At least in AKS we only set this on the agent, because that has the O(n^2) behavior where every agent watches every cilium endpoint and both grow with the size of the cluster.